src/components/mission/Mission.js

```diff
@@ -1,10 +1,19 @@
-import { useSelector } from 'react-redux';
-import { Container, Table, Button } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Container, Table, Button, Badge,
+} from 'react-bootstrap';
+import { joinMission } from '../../redux/missions/missions';
 
 const Mission = () => {
+  const dispatch = useDispatch();
   const missions = useSelector((state) => state);
   const missionsArray = Object.entries(missions)[0][1];
 
+  const handleClick = (e) => {
+    const { id } = e.target.parentNode.parentNode;
```

Comment:
I don't think they are a need to destructure id, you can make it a variable i.e: const id = e.target.parentNode.parentNode.